frontend/src/features/splReports/components/ReportForm.tsx

```diff
@@ -13,11 +17,26 @@ interface IReportFormProps {
  */
 const ReportForm: React.FunctionComponent<IReportFormProps> = ({ currentReport, snapshots }) => {
   const data: ISnapshot[] = snapshots ?? [];
+  const { getOptionsByType } = useCodeLookups();
+  const agencyItems = getOptionsByType(API.AGENCY_CODE_SET_NAME);
+  const agencyFilterOptions = React.useMemo(
+    () => (agencyItems || []).map(c => mapSelectOptionWithParent(c, agencyItems || [])),
+    [agencyItems],
+  );
+  const { snapshotFilter, setSnapshotFilter } = useSplReportContext();
+
   return (
     <Table
       name="spl report table"
       noRowsMessage="No Reports Available"
-      columns={columns}
+      filterable
+      filter={snapshotFilter as any}
```

Comment:
Filter as `any` also feels like a missed opportunity to create an interface.

Reply:
The table by default uses a filter of T = type of data display, so was a way to force it to use a custom filter type

Reply:
Addressed